packages/libs/core/src/locale.ts

```diff
@@ -37,21 +37,23 @@ export const getAcceptLanguageLocale = (
   routesManifest: RoutesManifest
 ) => {
   if (routesManifest.i18n) {
-    const locales = routesManifest.i18n.locales;
     const defaultLocale = routesManifest.i18n.defaultLocale;
+    const locales = new Set(
+      routesManifest.i18n.locales.map((locale) => locale.toLowerCase())
```

Comment:
It would be nice if this is already in lowercase but it's Next.js that generates the routes manifest, so we should try to preserve it as much as possible

Reply:
The case-correct versions are needed for route matching. But it would of course be possible to add lowercase versions to build manifest if desired.